### DISCUSSION TITLE
Assignments management

### DISCUSSION THREAD

**Discussion by komoornik** [Micke-K/IntuneManagement#120] [Ideas]
Is there a plan to make a nice bulk Assignments management tools? :)

Let's say we have a bunch of profiles we would like to select and then easily select included and excluded groups.

**Comment by Micke-K** [comment]
Hello!

Not the first time I get this question :). 

I've tried to avoid it. The reason is that it is not super easy since there are many different assignment types so it gets a bit complicated. The Policy Set object is supposed to take care of this and I was hoping MS would add support for all policy types but they haven't done that yet. 

I've focused on features not available in the Intune portal. Not sure how many out there that would use this feature. 

Cheers!
